RP Cloud Overview and Security doc (#1504)
redpanda-data/documentation: docs/deploy/deployment-option/cloud/security/cloud-security-architecture/index.mdx

---
title: Architecture
pagination_next: null
pagination_prev: null
---

<head>
    <meta name="title" content="Architecture  | Redpanda Docs"/>
    <meta name="description" content="Redpanda Cloud security adheres to the most stringent compliance rules and requirements."/>
</head>

- [Authorization](../cloud-security-architecture/authorization/cloud-authorization)

>    - An overview of Redpanda Cloud support for user and service authorizations using ACLs.

Reviewer: ```suggestion
   An overview of Redpanda Cloud support for user and service authorizations using ACLs.
```

- [Availability](../cloud-security-architecture/cloud-availability)

>     - An overview of Redpanda Cloud support for deploying clusters in single or multiple Availability Zones (AZs).

Reviewer: ```suggestion
    An overview of Redpanda Cloud support for deploying clusters in single or multiple availability zones (AZs).
```

- [Authentication](../cloud-security-architecture/cloud-authentication)

>    - An overview of Redpanda Cloud support for user and service authentication.

Reviewer: ```suggestion
   An overview of Redpanda Cloud support for user and service authentication.
```

- [Safety and Reliability](../cloud-security-architecture/cloud-safety-reliability)

>     - An overview of Redpanda Cloud support for Safety, reliability, and security, including auditing and testing, packaging, and self-regulation.

Reviewer: ```suggestion
    An overview of Redpanda Cloud support for safety, reliability, and security. This includes auditing and testing, packaging, and self-regulation.
```

- [Encryption](../cloud-security-architecture/cloud-encryption)

>    - An overview of Redpanda Cloud support for data at rest and data in transit encryption.

Reviewer: ```suggestion
   An overview of Redpanda Cloud support for data at rest and data in transit encryption.
```

- [Secrets](../cloud-security-architecture/secrets)

>    - An overview of Redpanda Cloud support for managing secrets, such as account credentials, keys, passwords, and certificates.

Reviewer: ```suggestion
   An overview of Redpanda Cloud support for managing secrets, such as account credentials, keys, passwords, and certificates.
```

- [Network Design, Ports, and Flows](../cloud-security-architecture/cloud-security-network)

>     - An overview of the Redpanda Cloud network design, including ports, and flows.

Reviewer: ```suggestion
    An overview of the Redpanda Cloud network design, including ports and flows.
```
